### DISCUSSION TITLE
[pybullet, slidebar] How to keep the values of the sliders consistent with the actions of the manipulator?

### DISCUSSION THREAD

**Discussion by borninfreedom** [bulletphysics/bullet3#4042] [Ideas]
Now I can add a slider through the addUserDebugParameter API or set the initial value of the slider. Now I want to control the movement of the manipulator through other methods, and then keep the value of the slider consistent with the movement of the manipulator. What should I do?

**Comment by erwincoumans** [comment]
It is not supported to programmatically set the value of a slider bar at the moment. It would require modifying the C++ code.

If I find time, I'll have a look implementing this.
